packages/cli-repl/package.json

```diff
@@ -22,5 +22,8 @@
     "mongosh-service-provider-server": "file:../service-provider-server",
     "mongosh-shell-api": "file:../shell-api",
     "read": "^1.0.7"
+  },
+  "devDependencies": {
+    "mocha": "^7.0.0"
```

Comment:
This should not be needed

Reply:
I feel like I should be able to run tests from this directory too, no? there are scripts defined above in package.json so I thought they should probably run.